### ISSUE TITLE
Improved BLAS/LAPACK guessing

### ISSUE THREAD

**Issue by prisae** [banesullivan/scooby#54]
If OK with @seberg we should include this
    https://gist.github.com/seberg/ce4563f3cb00e33997e4f80675b80953
instead of our often unsuccessful approach
```py
try:
    import mkl
    mkl.get_version_string()
except (ImportError, AttributeError):
    mkl = False

try:
    import numexpr
except ImportError:
    numexpr = False

# Get mkl info from numexpr or mkl, if available
if mkl:
    MKL_INFO = mkl.get_version_string()
elif numexpr:
    MKL_INFO = numexpr.get_vml_version()
else:
    MKL_INFO = False
```

What do you think @banesullivan ?

**Comment by seberg** [comment]
If it works for you, sure, go ahead. You might want to check out joblib and https://github.com/joblib/threadpoolctl (although maybe that has a different dependency). My approach does not really work on windows though, and I am not sure about mac.

**Comment by prisae** [comment]
Thanks @seberg. joblib would be another requirement, what we would like to avoid. If it doesn't work on Windows it might be an issue, but I cannot test. However, we could use our current approach and your approach. @banesullivan can test if it works on a mac. We'll let you know and definitely mention/cite you if we implement it! scooby is under the MIT license, would that be fine with you?

**Comment by seberg** [comment]
Honestly, please just use it as public domain, its not really anything special IMO.

**Comment by banesullivan** [comment]
@prisae, I'm all for it! I can test on Mac/Windows (I have Windows under a VM) once the PR is in and then we can decide on how to proceed.

Thanks for the good work in the gist, @seberg!

**Comment by prisae** [comment]
What is the status of this @banesullivan?

**Comment by prisae** [comment]
No, I cannot really remember why I opened it, or what triggered it. Just now I downloaded the gist to quickly try, and it actually returned `Unable to guess BLAS implementation`, while a `scooby.Report()` returns the MKL libray. I am sure there was a good reason due to something the gist can what scooby cannot, but I don't remember.

So I'll close this (if I ever remember I'll reopen).

I am also not sure if we improved something in the last two years already in this regard.

We might want to add some BLAS/LAPACK information at some point (maybe this was the reason?).
